### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by eko3alpha** [outcoldman/docker-stats-splunk#3]
I'm going to assume that this project is dead.  

````
Pulling splunk (outcoldman/docker-stats-splunk:latest)...
Pulling repository docker.io/outcoldman/docker-stats-splunk
ERROR: Error: image outcoldman/docker-stats-splunk:latest not found
````

I keep finding references for Splunk Light however nothing that works.  There is even a blog about it on Splunk's own blog but it still doesn't work.

http://blogs.splunk.com/2015/08/24/collecting-docker-logs-and-stats-with-splunk/

**Comment by alvarow** [comment]
I think it just moved to Splunk proper ... I have been trying: [6.5.2-monitor](https://hub.docker.com/r/splunk/splunk/) ... it mostly works, I am trying to run my own Splunk server on an instance instead of a container, and just the forwarder inside a container on ECS. I can't get the container names to resolve, most dashboard don't run, but I see what on the sources. I am thinking if I need anything other than `docker-app` on my Splunk instance (search header and indexer).

**Comment by outcoldman** [comment]
@alvarow @eko3alpha we have build a new more completed solutions to monitor docker and kubernetes https://www.outcoldsolutions.com please give it a try and let me know what you think.

**Comment by alvarow** [comment]
Who is we? and why are you spamming us?

**Comment by outcoldman** [comment]
@alvarow 
we - https://www.outcoldsolutions.com 
spamming? this is my repository under my account. I saw the question about the status of the project and provided cleared answer.

**Comment by alvarow** [comment]
I see now... perhaps you could be more clear about it, I tend no to follow spam... the reply certainly did fall into what I categorize as spam. Perhaps start with "Hey I opened I business around this, and other stuff similar to this".  I'll check it out.

thanks

**Comment by outcoldman** [comment]
@alvarow yeah, that would be more clear! understood, thank you for feedback :)
